GoogleChromeLabs/ps-analysis-tool
# Feature: Add info tab to bottom table.

## Description

This PR adds info tab to the bottom tray table and add information texts for advertiser and publishers.

## Relevant Technical Choices



## Testing Instructions

1. Open PA:EE tab and let the flow begin.
2. Click on any "i" icon
3. The tab should switch to "info" and information text appear as title.

## Additional Information:



## Screenshot/Screencast

<img width="938" alt="image" src="https://github.com/user-attachments/assets/38e0a69d-60c8-4793-8f10-01780a81972e" />


---

## Checklist






Partially addresses #857
